### PR TITLE
🎉 Oracle source: emit state messages more frequently

### LINKED ISSUE #10909: Add checkpointing to JDBC Source databases
## Summary

Created from Slack: https://airbytehq.slack.com/archives/C01MFR03D5W/p1640005429033600?thread_ts=1640005429.033600&cid=C01MFR03D5W

> when starting full sync on a huge table i can see that there is a long query in the DB(something like this):
> `SELECT <all columns> FROM <table>`
>
> But the table is 4TB, this way it will take days to finish the first query (i will probably fail before).. any way to configure the first sync to be divided into smaller chanks?

## Connector publication

| Connector | Old | New | PR |
| --- | --- | --- | --- |
| source-postgres | 0.4.40 | 0.4.41 | #14903 |
| source-postgres-strict-encrypt | ^ | ^ | ^ |
| source-mssql | 0.4.16 | 0.4.17 | #16261 |
| source-mssql-strict-encrypt | ^ | ^ | ^ |
| source-mysql | 0.6.7 | 0.6.8 | #16259 |
| source-mysql-strict-encrypt | ^ | ^ | ^ |
| source-oracle | 0.3.20 | 0.3.21 | #16238 |
| source-oracle-strict-encrypt | ^ | ^ | ^ |
| source-redshift | 0.3.13 | 0.3.14 | #16258 |
| source-snowflake | 0.1.19 | 0.1.20 | ^ |
| source-clickhouse | 0.1.12 | 0.1.13 | #16256 |
| source-clickhouse-strict-encrypt | 0.1.9 | ^ | ^ |
| source-cockroachdb | 0.1.16 | 0.1.17 | ^ |
| source-cockroachdb-strict-encrypt | ^ | ^ | ^ |
| source-db2 | 0.1.14 | 0.1.15 | ^ |
| source-db2-strict-encrypt | ^ | ^ | ^ |
| source-tidb | 0.2.0 | 0.2.1 | ^ |

`^` means it is the same as above.

### PR DESCRIPTION
**Pull request by tuliren**
## What
- Emit state messages more frequently to mitigate the impact of sync failures for large amount of data.
- Relate to https://github.com/airbytehq/airbyte/issues/10909.

## 🚨 User Impact 🚨
- No negative impact.
- When there is a large amount of data to sync and a failure in the middle, the next sync attempt won't need to start from scratch.

### CODE REVIEW

File: .github/workflows/slash-commands.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
         id: scd
         uses: peter-evans/slash-command-dispatch@v2
         with:
-          token: ${{ secrets.DAVINCHIA_PAT }}
+          token: ${{ secrets.SUPERTOPHER_PAT }}
```
Comment (tuliren) [inline .github/workflows/slash-commands.yml:21]: Need to revert this before merging.

**Review by ryankfu — APPROVED** [review]
LGTM 👍

**Pull request merged**